modules/services/taginfo.js

```diff
@@ -289,7 +289,7 @@ export default {
     values: function(params, callback) {
         // Exclude popular keys from values lookups.. see #3955
         var key = params.key;
-        if (key && _popularKeys[key]) {
+        if (key && _popularKeys[key] || key.slice(0,4)==='name') {
```

Comment:
[`String.prototype.startsWith`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith) is less error-prone than comparing a substring manually. A more thorough fix would catch any key if it’s just a localized version of any of the popular keys. A localized key looks like `baseKey:xy`, where _xy_ is a language code (not necessarily two characters long). I think what you’ve got is OK, but I’d recommend switching to `startsWith` and also requiring a colon, like `name:`, in case someone types in a key that doesn’t have it like [`name_disaster`](https://taginfo.openstreetmap.org/keys/name_disaster#overview) or [`name_base`](https://taginfo.openstreetmap.org/keys/name_base#overview).

Reply:
@1ec5 as you said that start with name: will cause to name that starting without colon  shows suggestion,  so do i change with this  key.slice(0,4)==='name'  to this =>  key.slice(0,5)==='name:'  ??

Reply:
> [`String.prototype.startsWith`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith) is less error-prone than comparing a substring manually. A more thorough fix would catch any key if it’s just a localized version of any of the popular keys. A localized key looks like `baseKey:xy`, where _xy_ is a language code (not necessarily two characters long). I think what you’ve got is OK, but I’d recommend switching to `startsWith` and also requiring a colon, like `name:`, in case someone types in a key that doesn’t have it like [`name_disaster`](https://taginfo.openstreetmap.org/keys/name_disaster#overview) or [`name_base`](https://taginfo.openstreetmap.org/keys/name_base#overview).

Now sir what I do :
1.can i just make the testcase for it and send PR for it and remaining thing should be the same??